frontend-vue/src/components/domains/ContextureDomainCardGrid.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div v-if="domains.length" class="mb-4">
+  <div v-if="domains.length" class="sticky top-0 mb-4 border border-x-0 border-t-0 border-blue-100 bg-white pb-2">
```

Comment:
```suggestion
  <div v-if="domains.length" class="sticky top-0 mb-4 border border-x-0 border-t-0 border-blue-100 bg-white py-2">
```

Just a nit to give it more space to breath 😄